Header header
VectorClass[] vector_classes
